ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly-4.19-upgrade-from-stable-4.18.yaml

```diff
@@ -794,6 +794,16 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-nutanix-ipi-compute-gpu
+- as: nutanix-ipi-preload-image-f60
```

Comment:
Oh, my fault, I re-consider the upgrade ci job, for the install workflow, it is just preload os image during installation, once cluster is installed, there is no any difference with non-preload cluster. So I guess we do not need to cover it in upgrade ci any more.

Reply:
Actually I even think the install workflow is only need to be covered in installer ci (openshift-verification-tests-master__installation-nightly-4.N.yaml), if we can develop a step to check it is using pre-load images to run installation, it would be better.